packages/graphql-toolbox/src/modules/EditorView/docexplorer/SearchResults.tsx

```diff
@@ -5,160 +5,148 @@
  *  LICENSE file in the root directory of this source tree.
  */
 
-import React, { ReactNode } from 'react';
-import { GraphQLSchema, GraphQLNamedType } from 'graphql';
+import React, { ReactNode } from "react";
```

Comment:
This file/component is from an external source, almost exclusively prettier updates.